proto/cluster.proto

```diff
@@ -207,21 +219,24 @@ message HeadGroupSpec {
   string image = 2;
   // Optional. The service type (ClusterIP, NodePort, Load balancer) of the head node
   string service_type = 3;
-  // Optional. The ray start params of head node group
-  map<string, string> ray_start_params = 4;
+  // Optional. Enable Ingress
+  // if Ingress is enabled, we might have to specify annotation IngressClassAnnotationKey, for the cluster itself, defining Ingress class
+  bool enableIngress = 4;
```

Comment:
This is a bit of breaking changed, and is avoidable. Can you add your `enableIngress` field as the last field and do not renumber the fields ?

Reply:
For HTTP it is not. Take a look at swagger - it is based on names. Its only braking for internals, preserving HTTP APIs

Reply:
I agree on the HTTP side, but nevertheless we should not be breaking the `gRPC` api.

Reply:
Done